src/DssSpell.sol

```diff
@@ -53,163 +57,74 @@ contract DssSpellAction is DssAction {
-
-        // Decrease the hop parameter for 7,884 seconds from 19,710 seconds to 11,826 seconds.
-        DssExecLib.setValue(MCD_FLAP, "hop", 11_826);
-
-        // Increase the bump parameter for 25,000 DAI from 50,000 DAI to 75,000 DAI.
-        DssExecLib.setValue(MCD_VOW, "bump", 75 * THOUSAND * RAD);
-
-        // ---------- Approve HVBank (RWA009-A) Dao Resolution ----------
-        // Forum: https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264/24
-
-        // Approve HVBank (RWA009-A) Dao Resolution with IPFS hash QmStrc9kMCmgzh2EVunjJkPsJLhsVRYyrNFBXBbJAJMrrf
-        // Note: see `dao_resolutions` variable declared above
+        // ---------- Increase the GSM Pause Delay ----------
+        // Forum: https://forum.makerdao.com/t/gsm-pause-delay-increase-proposal/23929
+        // Poll: https://vote.makerdao.com/polling/QmcLsYwj
+
+        // Increase the GSM Pause Delay by 14 hours, from 16 hours to 30 hours -----
+        PauseLike(MCD_PAUSE).setDelay(30 hours);
+
+        // ---------- Spark MetaMorpho Vault DDM ----------
+        // Forum: https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006
+        // Forum: https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006/8
+
+        // DDM DC-IAM Parameters: line: 1 billion DAI
+        DssExecLib.setIlkAutoLineDebtCeiling("DIRECT-SPARK-MORPHO-DAI", 1 * BILLION);
+
+        // ---------- Add the following ilks to LINE_MOM ----------
+        // Forum: https://forum.makerdao.com/t/gov12-1-2-bootstrapping-edit-proposal-gov10-2-3-1a-edit/24005
+        // Poll: https://vote.makerdao.com/polling/QmZsAM36
+
+        // ETH-A
+        LineMomLike(LINE_MOM).addIlk("ETH-A");
+
+        // ETH-B
+        LineMomLike(LINE_MOM).addIlk("ETH-B");
+
+        // ETH-C
+        LineMomLike(LINE_MOM).addIlk("ETH-C");
+
+        // WSTETH-A
+        LineMomLike(LINE_MOM).addIlk("WSTETH-A");
+
+        // WSTETH-B
+        LineMomLike(LINE_MOM).addIlk("WSTETH-B");
+
+        // WBTC-A
+        LineMomLike(LINE_MOM).addIlk("WBTC-A");
+
+        // WBTC-B
+        LineMomLike(LINE_MOM).addIlk("WBTC-B");
+
+        // WBTC-C
+        LineMomLike(LINE_MOM).addIlk("WBTC-C");
 
         // ---------- Spark Proxy Spell ----------
-        // Forum: https://forum.makerdao.com/t/mar-6-2024-proposed-changes-to-sparklend-for-upcoming-spell/23791/
-        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-11-under-sta-article-3-3/23910
-        // Poll: https://vote.makerdao.com/polling/QmQAtb17
-        // Poll: https://vote.makerdao.com/polling/QmYYoAMe
+        // Forum: https://forum.makerdao.com/t/mar-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/23918
+        // Poll: https://vote.makerdao.com/polling/QmdjqTvL
+        // Poll: https://vote.makerdao.com/polling/QmaEqEav
+        // Poll: https://vote.makerdao.com/polling/QmbCWUAP
```

Comment:
The Spark spell also has a part for adjusting the Morpho supply caps, so you can also add this link:
https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006